src/fiatconnect/index.test.ts

```diff
@@ -29,4 +30,34 @@ describe('FiatConnect helpers', () => {
       expect(providers).toEqual([])
     })
   })
+
+  describe('addNewFiatAccount', () => {
+    // TODO: unskip after integration with FC SDK
+    // eslint-disable-next-line jest/no-disabled-tests
+    it.skip('returns a fiat account info with fiat account id on success', async () => {
```

Comment:
Instead of skipping and having a todo can you run the test and check for the thrown `'Not implemented'` error? That way when its implemented the test will fail and we will be sure not to miss this.